### PR TITLE
Fix license header script and update headers

### PR DESCRIPTION
**Pull request by ChristophWurst**
This hasn't been done in a while. Thus the large diff.

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: apps/accessibility/appinfo/app.php [hunk 1]
```diff
@@ -18,7 +22,7 @@
  * GNU Affero General Public License for more details.
  *
  * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (skjnldsv) [inline apps/accessibility/appinfo/app.php]: Why is there a double space, xD

**Comment by skjnldsv** [comment]
Should we have a test for it? :p
And a bot?

**Comment by rullzer** [comment]
No bot.
We should just remember to run it once before a release or something.

**Review by rullzer — APPROVED** [review]

**Comment by skjnldsv** [comment]
There was some weird changes, like this one
https://github.com/nextcloud/server/blob/5bf3d1bb384da56adbf205752be8f840aac3b0c5/apps/settings/lib/Controller/UsersController.php#L10

**Comment by kesselb** [comment]
> There was some weird changes, like this one

:fearful: 

```
grep -c -r --include \*.php  "is free software: you can redistribute it and\/or modify" . | grep -v -E ':0|:1'
./build/license.php:3
./apps/settings/lib/Controller/UsersController.php:2
./apps/settings/lib/Controller/ChangePasswordController.php:2
./apps/text/lib/Controller/DirectSessionController.php:2
./apps/text/lib/Controller/WorkspaceController.php:2
./apps/recommendations/lib/Service/RecentlyCommentedFilesSource.php:2
```

Seems to affect only a few files.

**Comment by kesselb** [comment]
@ChristophWurst could you have a look at the broken files?

**Pull request merged**